pkg/trojan/message_test.go

```diff
@@ -124,6 +125,23 @@ func TestWrapFail(t *testing.T) {
 	}
 }
 
+func TestWrapTimeout(t *testing.T) {
+	m := newTestMessage(t)
+
+	// a large target will take more than MinerTimeout seconds, so timeout error will be triggered
+	buf := make([]byte, swarm.SectionSize)
+	_, err := rand.Read(buf)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
sorry. i will like to stick to my original style, since i thin that is more readable.
if this is part of the go-coding guideline, then please quote the place so that i know.

Reply:
no problem :+1: